resources/core/templates/tests/rbac-external-solution.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or .Values.test.external_solution.event_bus.enabled .Values.test.external_solution.event_mesh.enabled }}
+{{- if .Values.test.external_solution.event_mesh.enabled }}
```

Comment:
Will you also do a cleanup on `test-infra`? If so, could you please remove this line here within the same PR? Thanks!
https://github.com/kyma-project/test-infra/blob/master/prow/scripts/cluster-integration/kyma-gke-compass-integration.sh#L226

Thanks!

Reply:
ok, will do

Reply:
🙇 Thank you!